sympy/vector/vector.py

```diff
@@ -40,6 +40,34 @@ def components(self):
         # subclass of Vector the instance belongs to.
         return self._components
 
+    @property
+    def projections(self):
+        """
+        Returns the components of this vector but the output includes
+        also zero values components.
+
+        Examples
+        ========
+
+        >>> from sympy.vector import CoordSys3D
+        >>> C = CoordSys3D('C')
+        >>> v = 3*C.i + 4*C.j + 5*C.k
+        >>> v.projections
+        {C.i: 3, C.j: 4, C.k: 5}
+        >>> v = C.x*C.y*C.z*C.i
+        >>> v.projections
+        {C.i: C.x*C.y*C.z, C.j: 0, C.k: 0}
+        >>> v = Vector.zero
+        >>> v.projections
+        0
```

Comment:
I have problem here, because `Vector.zero` does not exist in any coordinate system so I can't get information about base vectors